claas/api.py

```
@@ -0,0 +1,345 @@
+    5. Updates LoRA parameters
+    6. Saves the updated LoRA back to S3
+
+    Returns the new LoRA URI and training metrics.
+    """
+    try:
+        # Validate LoRA exists
+        if not lora_exists(request.lora_uri):
+            raise HTTPException(
+                status_code=404,
+                detail=f"LoRA not found at {request.lora_uri}",
+            )
```

Ruff (0.14.14)

[warning] Abstract `raise` to an inner function

(TRY301)